lib/store/index.js

```diff
@@ -31,9 +31,18 @@ export class Store {
   @observable editor = atom.workspace.getActiveTextEditor();
   @observable grammar: ?atom$Grammar;
   @observable configMapping: Map<string, ?mixed> = new Map();
+  globalMode: boolean = Boolean(atom.config.get("Hydrogen.globalMode"));
```

Comment:
This mode won't change while hydrogen is activated. I was afraid live updating would cause too many problems and most people will keep this one or the other most of the time anyway